modules/mono/glue/GodotSharp/GodotSharp/Core/Transform2D.cs

```diff
@@ -307,7 +309,21 @@ public Transform2D Rotated(real_t angle)
         }
 
         /// <summary>
-        /// Scales the transform by the given scaling factor, using matrix multiplication.
+        /// Rotates the transform by <paramref name="angle"/> (in radians).
+        /// The operation is done in the local frame, equivalent to
+        /// multiplying the matrix from the right.
+        /// </summary>
+        /// <param name="angle">The angle to rotate, in radians.</param>
+        /// <returns>The rotated transformation matrix.</returns>
+        public Transform2D RotatedLocal(real_t phi)
+        {
+            return new Transform2D(phi, new Vector2()) * this;
+        }
```

Comment:
```suggestion
        public Transform2D RotatedLocal(real_t angle)
        {
            return new Transform2D(angle, new Vector2()) * this;
        }
```

Reply:
Indeed, I missed that replacement of `phi` => `angle`. I've committed it locally to allow for `--amend` instead of extra commit.